### ISSUE TITLE
Replies to a forum thread you are following not consistently showing up on Messages page

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#1538]
Needs investigation! I am looking into it to see if I can get the repro steps down, then I will discuss with @mewtaylor.

This may be working as intended, I'm not totally sure yet. But I am logging it because after discussing it & trying it out with Scratchers in the forums, I did see the issue myself, and it is at least somewhat confusing behaviour.

I (as stymphalianbirb) did not get notifications about new replies to a topic I was following, but some of my test accounts did. I am wondering if it has to do with whether the reply quotes you or is directly after your own post or something...

Will test now and update with more info.

**Comment by jwzimmer** [comment]
**Trying to repro on Staging**

Neither abcdefky nor jwzimmer1990 received a new notification about this post, although neither of them had viewed the forum page since it was made (I would not expect abcdefky to get a notification, since they made the post, but I would expect jwzimmer1990 to get a notification about it).
<img width="981" alt="screen shot 2017-09-12 at 5 01 50 pm" src="https://user-images.githubusercontent.com/1320456/30348101-3403c8a2-97dc-11e7-8120-04b11579229e.png">

**Comment by jwzimmer** [comment]
As jwzimmer1990, I have not received a notification about this new reply in a topic I'm following. I have **not** viewed the messages page since the reply was made.

<img width="1000" alt="screen shot 2017-09-13 at 10 45 50 am" src="https://user-images.githubusercontent.com/1320456/30383927-d2e3bc72-9870-11e7-8124-3f1a6dc0e85b.png">

**Comment by jwzimmer** [comment]
@mewtaylor & I looked at this issue together, and repro'd it. @mewtaylor is investigating.

**Comment by jwzimmer** [comment]
Current theory is that it has to do with outdated workers. @mewtaylor is working on it.

**Comment by mewtaylor** [comment]
@jwzimmer I think this is fixed now (or at least the celery issue is fixed). I will leave it to you to keep open/closed though, depending on if you're still seeing problems?

**Comment by jwzimmer** [comment]
@towerofnix oh no! Thanks for letting us know.

**Comment by GreenBayRules** [comment]
Yeah this is happening to me also in this thread: https://scratch.mit.edu/discuss/topic/278050/

**Comment by VutonDesign** [comment]
Same here now, None of my stickies get notifications.
